pkg/schema/load/panel.go

```diff
@@ -58,7 +58,10 @@ func mapPanelModel(id string, vcs schema.VersionedCueSchema) cue.Value {
 		panelSchema: maj: in.v.maj
 		panelSchema: min: in.v.min
 		options: in.model.PanelOptions
-		fieldConfig: defaults: custom: in.model.PanelFieldConfig
+		fieldConfig: defaults: custom: {}
+		if in.model.PanelFieldConfig != _|_ {
```

Comment:
I am unfamiliar with this wizardry :shrug:

Reply:
`_|_` is [bottom](https://cuelang.org/docs/concepts/logic/#cue-types). Comparing to bottom like this can get a little weird, but the effect here is checking whether or not `in.model.PanelFieldConfig` exists before creating a reference to it.